riposte-spi/src/main/java/com/nike/riposte/server/config/ServerConfig.java

```diff
@@ -390,12 +390,11 @@ default int maxOpenIncomingServerChannels() {
     /**
      * @return The maximum allowed request size in bytes. If netty receives a request larger than this then it will
      * throw a {@link io.netty.handler.codec.TooLongFrameException}.
-     * This value is an integer, so the max you can set it to is {@link Integer#MAX_VALUE}, which corresponds to 2^31-1,
```

Comment:
I would leave the note about max size in here (but since you're fixing the `NOTE` that line should obviously go).